docs/websockets.md

```diff
@@ -10,13 +10,13 @@ To communicate between our different services we use websockets. Every AWX node
 
 Inside AWX we use the `emit_channel_notification` function which places messages onto the queue. The messages are given an explicit event group and event type which we later use in our wire protocol to control message delivery to the client.
 
-### Broadcast Backplane
+### Relay Backplane
 
-Previously, AWX leveraged RabbitMQ to deliver Ansible events that emanated from one AWX node to all other AWX nodes so that any client listening and subscribed to the Websockets could get events from any running playbook. We are since moved off of RabbitMQ and onto a per-node local Redis instance. To maintain the requirement that any Websocket connection can receive events from any playbook running on any AWX node we still need to deliver every event to every AWX node. AWX does this via a fully connected Websocket backplane. 
+Previously, AWX leveraged RabbitMQ to deliver Ansible events that emanated from one AWX node to all other AWX nodes so that any client listening and subscribed to the Websockets could get events from any running playbook. We are since moved off of RabbitMQ and onto a per-node local Redis instance. To maintain the requirement that any Websocket connection can receive events from any playbook running on any AWX node we still need to deliver every event to every AWX node. AWX does this via a fully connected Websocket backplane.
```

Comment:
As part of this feature, we no longer send every event to every AWX node.

We can also drop the comments regarding rabbitMQ

I would replace this paragraph with something like,

"A core feature of AWX is that any websocket connection can receive events from any running playbook, regardless of the control node that is handling those events. Each control node runs a service called `wsrelay`. This service maintains an active connection to each running web service, forming a websocket backplane. The web services are powered by Redis-based Django channels to handle websocket messages. As jobs run and events are processed, the control node sends events _only_ to web services that subscribe for those job events."